_posts/2013-02-01-client_libraries.md

```diff
@@ -439,6 +439,19 @@ production? Tell us about it on the [mailing list][mailing_list] or Twitter [@im
     <td></td>
     <td></td>
   </tr>  
+  <tr class="warning">
+    <td><a href="https://github.com/aohan237/asyncnsq">asyncnsq</a></td>
+    <td>Python/Http</td>
+    <td><i class="fa fa-check"></i></td>
+    <td><i class="fa fa-check"></i></td>
+    <td></td>
```

Comment:
asyncnsq supports `lookupd_http_addresses` so I think it more or less does support Discovery?

Reply:
yea..

i just dont know how to judge support discovery ,so just leave it...